libgstc/python/Makefile.am

```diff
@@ -1,2 +1,13 @@
-install-am:
-	@sudo -H pip3 install .
+all-local:
+	(cd $(srcdir); python3 setup.py build --version=$(PACKAGE_VERSION))
```

Comment:
build should be in the builddir. Can something like this be made?
```
(cd $(srcdir); python3 setup.py $(builddir)/build --version=$(PACKAGE_VERSION))
```

Reply:
`build` is not the buildir, but rather the command for `setup.py`. `setuptools` doesn't have an option by default to change the buildir, it will create build and .pybuild/ in the cwd.